### PR TITLE
Update quiz.txt first question

### PR DESCRIPTION
**Pull request by tomkidoz**
I answered first **question**

### CODE REVIEW

File: quiz.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-2+3=?
```
Comment (tomkidoz) [inline quiz.txt:1]: I chose this becoz I love it

Comment (tomkidoz) [reply quiz.txt:1]: and I may have other question

Comment (unclecode) [reply quiz.txt:1]: okay choose another

**Review by unclecode — CHANGES_REQUESTED** [review]
make it in word

**Comment by tomkidoz** [comment]
a help?

**Comment by unclecode** [comment]
like four instead of 4

**Review by tomkidoz — COMMENTED** [review]
so just word?

**Review by unclecode — COMMENTED** [review]
and nothing

**Comment by tomkidoz** [comment]
Ok I am on it

File: quiz.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-2+3=5
+2+3=five
```
Comment (unclecode) [inline quiz.txt]: capitalisation

Comment (tomkidoz) [reply quiz.txt]: Oh Ok I will do it but would u make it review

Comment (unclecode) [reply quiz.txt]: Ok now we need ro review it

**Review by unclecode — APPROVED** [review]
I like it now

**Pull request merged**